emails/templates/emails/first_time_user.html

```diff
@@ -0,0 +1,411 @@
+    }
+    #how-relay-works-section {
+      padding: 0 48px 32px 48px; 
+    }
+    #get-relay-premium-section .content {
+      width: 50%;
+    }
+    #get-relay-premium-section .image img {
+      max-width: 117px;
+    }
+    #get-relay-premium-section .cta a { 
+      font-size: 16px;
+    }
+    @media only screen and (max-width: 768px) {
+      h1.hero-title {
+        font-size: 32px;
+        padding: 0 0 16px 48px;
+      }
+      p.hero-text {
+        font-size: 14px;
+      } 
+      #get-relay-premium-section .image img {
+        max-width: 90px;
+      }
+      #get-relay-premium-section .cta a { 
+        font-size: 14px;
+      }
+    }
+    @media only screen and (max-width: 600px) {
+      h1.hero-title {
+        font-size: 38px;
+        padding: 32px 0 16px 32px
+      }
+      p.hero-text {
+        font-size: 20px;
+        padding: 0 32px 
+      } 
+      #hero-section td {
+        width: 100%;
+        display: table-footer-group;
+      }
+      #hero-section img {
+        width: 90%;
+        margin: 0 auto;
+        display: block;
+      }
+      #what-is-relay-section td {
+        padding: 48px 32px 0 32px;
+      }
+      .separator {
+        width: 80%;
+      }
+      #how-relay-works-section {
+        padding: 0 32px 32px 32px; 
+      }
+      #how-relay-works-section td img {
+        min-width: 20px;
+      }      
+      #get-relay-premium-section {
+        width: 90%;
+      }
+      #get-relay-premium-section td {
+        width: 100%;
+        display: block;
+        margin: 0 auto;
+        text-align: center;
+      }
+      #get-relay-premium-section .content {
+        width: 80%;
+      }
+      #get-relay-premium-section .content .title {
+        padding: 15px 0;
+      }
+      #get-relay-premium-section .cta a {
+        margin: 20px auto;
+      }
+    } 
+     
+  </style> 
+</head>
+
+<body style="margin: 0;padding: 0;-webkit-text-size-adjust: 100%;background-color: #ffffff;color: #000000">
+  <div style="display:none; font-size: 1px; line-height: 1px; max-height: 0px; max-width:0px; opacity: 0; overflow: hidden;">
+      {% ftlmsg 'first-time-user-email-preheader' %}
+  </div>
+  <table style="vertical-align: top;min-width: 320px;margin: 0 auto;width:100%" cellpadding="0" cellspacing="0">
+    <tbody>
+      <tr style="vertical-align: top">
+        <td style="word-break: break-word;border-collapse: collapse ;vertical-align: top">
+
+          <!-- header -->
+          <table id="header-section" width="100%" border="0" style="max-width: 850px; display: block;margin:0 auto;">
+              <tbody style="display: block;">
+                <tr style="display: block;">
+                  <td style="display: block;"> 
+                    <img align="center" border="0" src="{{ SITE_ORIGIN }}/static/images/email-images/first-time-user/relay-wordmark.png" alt="" title="" style="height: auto;float: none;width: 100%;max-width: 125px; margin: 10px auto; display: block;" /> 
```

Comment:
The images are in the PR, but don't load for me locally. Why aren't we using `{% static "images/email-images/first-time-user/relay-wordmark.png" %}`?